libs/martech/martech.js

```diff
@@ -76,6 +76,18 @@ const handleAlloyResponse = (response) => {
     .filter(Boolean);
 };
 
+const getPropositionsInfo = (response) => {
```

Comment:
Do you have to modify `response.propositions` since the only thing being changed is removal of the `items` array?  Can it be sent to `satellite.track` with `items`?